Tests/ConcordiumSwiftSdkTests/ConcordiumHdWalletTest.swift

```diff
@@ -0,0 +1,187 @@
+@testable import ConcordiumSwiftSdk
+import CryptoKit
+import XCTest
+
+final class ConcordiumHdWalletTest: XCTestCase {
+    let TEST_SEED_1 = "efa5e27326f8fa0902e647b52449bf335b7b605adc387015ec903f41d95080eb71361cbc7fb78721dcd4f3926a337340aa1406df83332c44c1cdcfe100603860"
```

Comment:
Why does this name end with 1?

Reply:
Because that's the name used in https://github.com/Concordium/concordium-node-sdk-js/blob/main/packages/sdk/test/ci/HdWallet.test.ts#L186 where it was ported from. Feel free to change it.